packages/client/src/components/update-password-form.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+export default function UpdatePasswordForm() {
+  return (
+    <Form action="/update_password" method="post">
+      <label>Username</label> <input name="username" type="text" />
+      <label>Current Password</label> <input name="username" type="password" />
+      <label>New Password</label> <input name="password" type="password" />
```

Comment:
These 3 input fields need to have different `name`s. This is the prop that we end up reading on the backend.